test/blackbox-tests/test-cases/melange/multilib.t/run.t

```diff
@@ -0,0 +1,5 @@
+Compilation using melange with a library
+The library in the test includes tens of files to make sure cmjs are read after they are written.
```

Comment:
Why do we need so many files for this test? If it's some sort of benchmark, then it shouldn't be part of the test suite.

Reply:
This test was added to check some bug where cmjs were not added as deps when one lib depends on another. I originally thought the error would only happen with large builds, but I just realized it happens with single module libraries 🤦  fixed in https://github.com/jchavarri/dune/pull/5/commits/5da395db6ff91afb8025721de69ca2a16dd6d48d.